src/RamAnalysis.h

```diff
@@ -40,7 +40,7 @@ class RamAnalysis {
     virtual void run(const RamTranslationUnit& translationUnit) = 0;
 
     /** @brief Print the analysis result in HTML format */
-    virtual void print(std::ostream& os) const {}
+    virtual void print(std::ostream&) const {}
```

Comment:
Keep the commented variable name in the base class.
```suggestion
    virtual void print(std::ostream& /* os */) const {}
```